### PR TITLE
Admin invoices

### PR DESCRIPTION
**Pull request by kohyoungheon**
User Stories 32-36 are completed.
Features test coverage is 100%
Model test coverage is 100%

### CODE REVIEW

File: app/controllers/invoices_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class InvoicesController < ApplicationController
+  def update
+    invoice = Invoice.find(params[:id])
+    status = params[:status]
```
Comment (cariperi) [inline app/controllers/invoices_controller.rb:4]: It's super minor, but do we need to create a "status" variable here on line 4?  Could we pass params[:status] in to the argument on line 5 (e.g. invoice.update(status: params[:status])?

File: app/models/invoice.rb [hunk 2]
```diff
@@ -9,7 +9,20 @@ class Invoice < ApplicationRecord
...
 
+  def self.order_by_id
+    order(:id)
+  end
+
   def customer_name
     customer.first_name + " " + customer.last_name
   end
+
+  def convert_created_at
```
Comment (cariperi) [inline app/models/invoice.rb:20]: nice! I also had a to write a helper to format the invoice created at date, and yours here is much cleaner so I will default to that one when merging my new work in!

**Review by cariperi — COMMENTED** [review]
This looks awesome to me! Great job with test data setup, and nice use of helper methods, queries, and some path helpers throughout the tests / views / controllers.   

I left a couple of comments, leaving this open now just so you can see them but happy to merge once you confirm.

File: app/controllers/invoices_controller.rb (hunk 1 above)
Comment (kohyoungheon) [reply app/controllers/invoices_controller.rb:4]: Fixed! Refactored!

**Review by cariperi — APPROVED** [review]

**Review by cariperi — APPROVED** [review]

**Pull request merged**